### PR TITLE
Frontend integration

### PR DESCRIPTION
**Pull request by joonalillfors**

### CODE REVIEW

File: client/src/index.js [hunk 1]
```diff
@@ -45,9 +78,55 @@ class MainView extends React.Component {
...
+    <div className={css('feedback-app-main-container')}>
+      <MainView />
+    </div>,
+    feedbackAppRoot()
+  )
+}
+
+initialize()
+document.addEventListener('DOMContentLoaded', initialize)
+window.addEventListener('load', initialize)
```
Comment (HenrikHelenius) [inline client/src/index.js:132]: This is not reliable, google or read e.g. https://stackoverflow.com/questions/588040/window-onload-vs-document-onload

Does not have to be changed now but maybe rethought at some point?

File: client/src/meta/env.meta.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
 const meta = require('./meta-util')
 
-const staticUrl = process.env.STATIC_URL || 'localhost:8080'
+const staticUrl = process.env.STATIC_URL || 'http://localhost:8080'
 
-module.exports = meta.json({ staticUrl })
+const apiUrl = process.env.API_URL || 'http://localhost:8080/api'
```
Comment (HenrikHelenius) [inline client/src/meta/env.meta.js:5]: TODO later to https

File: server/src/routes/users.js [hunk 3]
```diff
@@ -24,7 +24,10 @@ router.post('/', catchErrors(async (req, res) => {
   await attempt(async () => {
     const id = uuid()
     const secret = uuid(30)
+
     await addUser({ id, name, secret })
+    console.log(`id: ${id}`)
```
Comment (HenrikHelenius) [inline server/src/routes/users.js]: ask samuli

File: client/src/index.js (hunk 1 above)
Comment (bqqbarbhg) [reply client/src/index.js:132]: Shouldn't hurt as `initialize()` works only on the first call

File: client/src/meta/env.meta.js (hunk 2 above)
Comment (bqqbarbhg) [reply client/src/meta/env.meta.js:5]: Localhost will always be `http://` https can be added through `STATIC_URL` environment variable

File: server/src/routes/users.js (hunk 3 above)
Comment (bqqbarbhg) [reply server/src/routes/users.js]: Yea this `console.log` doesn't belong here

**Review by bqqbarbhg — APPROVED** [review]
SHIP IT

**Pull request merged**